test/frontend-integration-test/Dockerfile

```diff
@@ -1,18 +1,8 @@
-FROM selenium/standalone-chrome
+FROM gcr.io/ml-pipeline-test/selenium-standalone-chrome-gcloud-nodejs:v20190507-0.1.19-19-g0b8b4dd7-dirty-20a6f9
```

Comment:
Is this image built manually? How often do we need to rebuild this?

Reply:
> Is this image built manually?

Yes. It's build using the provided Makefile which just builds the Dockerfile and pushes the image.

>How often do we need to rebuild this?

Only if you want to upgrade Selenium, Chrome, Node etc.